pkg/omf/cli/omf.install.fish

```diff
@@ -36,7 +36,7 @@ function omf.install -a type_flag name_or_url
   if test -e $OMF_PATH/db/$parent_path/$name_or_url
     set target $parent_path/$name_or_url
   else
-    set -l local_name (basename $name_or_url)
+    set -l local_name (basename $name_or_url | sed "s/pkg-//;s/plugin-//;s/theme-//")
```

Comment:
I have added the anchor element, but your example on number 3 is still not going to work. :+1: 

If you find a way to use `|` without using extended mode, let me know!